libs/angular-integration-interface/mocks/app-state-service-mock.ts

```diff
@@ -21,7 +21,7 @@ export class AppStateServiceMock {
   globalLoading$ = new FakeTopic(false)
   currentMfe$ = new FakeTopic({ mountPath: '/', remoteBaseUrl: '.', baseHref: '/', shellName: 'test' })
   currentPage$ = new FakeTopic<PageInfo | undefined>(undefined)
-  currentPortal$ = new FakeTopic<Workspace>({ baseUrl: '/', microfrontendRegistrations: [], portalName: 'Test portal', workspaceName: 'Test portal' })
-  currentWorkspace$ = new FakeTopic<Workspace>({ baseUrl: '/', microfrontendRegistrations: [], portalName: 'Test workspace', workspaceName: 'Test workspace' })
+  currentPortal$ = new FakeTopic<Workspace>({ baseUrl: '/', workspaceName: 'Test portal' })
+  currentWorkspace$ = new FakeTopic<Workspace>({ baseUrl: '/', workspaceName: 'Test workspace' })
```

Comment:
We should not change the mock because it can maybe used for tests of the apps. In the definition of the topic data structures we can not remove deprecated stuff because it is a protocol which is maybe also read by older versions of the lib